matrix-sdk/src/main/java/org/matrix/androidsdk/rest/model/Event.java

```diff
@@ -671,6 +671,9 @@ public boolean isSent() {
                 if (null != videoMessage.getUrl()) {
                     urls.add(videoMessage.getUrl());
                 }
+                if (null != videoMessage.getThumbnailUrl()) {
+                    urls.add(videoMessage.getThumbnailUrl());
+                }
```

Comment:
Location message also contains thumbnail url. Consider adding it?

```
            } else if (Message.MSGTYPE_LOCATION.equals(msgType)) {
                LocationMessage locationMessage = JsonUtils.toLocationMessage(getContent());

                if (null != locationMessage.thumbnail_url) {
                    urls.add(locationMessage.thumbnail_url);
                }
```

Reply:
Thx I added this url, 
but I'm not sure the location messages are correctly handled...